uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/LeaveRequest.php

```diff
@@ -41,7 +41,19 @@ function civicrm_api3_leave_request_create($params) {
   $values = [];
   _civicrm_api3_object_to_array($leaveRequest, $values[$leaveRequest->id]);
 
-  return civicrm_api3_create_success($values, $params, null, 'create', $leaveRequest);
+  $extraParams = ['from_email_configured' => true];
+  if(!civicrm_api3_leave_request_get_from_email_for_notifications($leaveRequest)) {
+    $extraParams['from_email_configured'] = false;
```

Comment:
I've checked the ticket, but there's no mention there of this `from_email_configured` param. What is it for? Is it going to be used by FE so that they can notify the user?

Reply:
Yes. It will be used by FE to display the popup error.